dap_prinz_green_jobs/utils/bert_vectorizer.py

```diff
@@ -23,7 +24,7 @@ def get_embeddings(
         dict: The sentence (key) and the embedding (value)
     """
 
-    bert_model = BertVectorizer(verbose=True, multi_process=True).fit()
+    bert_model = BertVectorizer(verbose=True, multi_process=False).fit()
```

Comment:
@india-kerle FYI I know we've been copying this code for a while, but `multi_process=True` seems to be a lot slower than `multi_process=False`!! I'm not quite sure why. But I tried it using the gpu on EC2, and on my own computer, and for both times `multi_process` is much faster when set to False! Weird because surely multiprocessing is sometimes a good thing?!

Worth noting since we should probably change this in ojd_daps_skills etc!